1-org/scc_notification.tf

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/******************************************
+  SCC Notification
+*****************************************/
+
+locals {
+  description = "SCC Notification for all active findings"
+}
+
+
+resource "google_pubsub_topic" "scc_notification_topic" {
+  name    = "top-scc-notification"
+  project = module.scc_notifications.project_id
+}
+
+resource "google_pubsub_subscription" "scc_notification_subscription" {
+  name    = "sub-scc-notification"
+  topic   = google_pubsub_topic.scc_notification_topic.name
+  project = module.scc_notifications.project_id
+
+  message_retention_duration = "1200s"
```

Comment:
Are we giving opinionated best values for constants like `message_retention_duration` `retain_acked_messages` `ack_deadline_seconds ` `expiration_policy` etc from an SCC pov? Otherwise I think we should just leave these to provider defaults.

Reply:
Agree with this, unless these are the values from documentation / white paper then we should use provider defaults